mock/src/main.rs

```diff
@@ -0,0 +1,137 @@
+use std::{fs::File, io::Write, path::PathBuf, time::Instant};
+
+use clap::Parser;
+use crypto::{hash::rpo::RpoDigest as Digest, FieldElement, Word};
+use miden_objects::{
+    builder::DEFAULT_ACCOUNT_CODE,
+    mock::{Immutable, MockChain, OnChain},
+};
+use rand::SeedableRng;
+use rand_pcg::Pcg64;
+
+#[derive(Parser, Debug)]
+struct Args {
+    /// Value used to seed the PRNG.
+    #[arg(long)]
+    seed: Option<u64>,
+
+    /// Output file
+    output: PathBuf,
+}
+
+fn seed_to_word(seed: &str) -> Word {
+    let seed_bytes = hex::decode(seed).unwrap();
+    let data = unsafe { FieldElement::bytes_as_elements(&seed_bytes) }.unwrap();
+    let seed: Word = [data[0], data[1], data[2], data[3]];
+
+    seed
+}
```

Comment:
minor nit: I'd move this down (right above tests) and put it into our usual `HELPER FUNCTIONS` section.

Reply:
changed